docs/encyclopedia/temporal-nexus-api-registry.mdx

```diff
@@ -0,0 +1,20 @@
+---
+id: nexus-api-registry
+title: Nexus API Registry
+sidebar_label: Nexus API Registry
+description: Learn how to use the Nexus API Registry to manage Nexus Endpoints. 
+toc_max_heading_level: 4
+slug: /temporalnexus/api-registry
+keywords:
+  - API registry
+  - service documentation
+tags:
+  - distributed systems
+  - integration
+  - temporal
+---
+
+### Nexus API Registry {#nexus-api-registry}
+
+The Nexus API Registry is an account-scoped list of Nexus `Endpoints` available within an account.
+Endpoints descriptions are rendered as markdown, and can include Service and Operation descriptions, and input/output types, so consumers know how to use the Services and Endpoint offers.
```

Comment:
```suggestion
Endpoints descriptions are rendered as markdown, and can include Service and Operation descriptions, and input/output types, so consumers know how to use the Services that the Endpoint offers.
```

Guessing at this meaning, not sure if it's correct

Reply:
I'm going to keep 'and' for now.